scripts/ts-generate-definition.js

```diff
@@ -12,26 +12,28 @@ fs.readFile(clientFile, 'utf8', (err, data) => {
     process.exit(2);
   }
 
-  let clientMethods = data.split('\n')
-    .filter(line => line.match(/^  \w.*(.*): .*{$/))
-    .map(line => line
-      .replace(/ {$/, '')
-      .replace(': ?', '?: ')
-    ).join('\n');
+  const clientMethods = data
+    .split('\n')
+    .filter(line => line.match(/^ {2}(?!static create)\w.*(.*): .*{$/))
+    .map(line => line.replace(/ {$/, '').replace(': ?', '?: '))
+    .join('\n');
 
   fs.readFile(typesFile, 'utf8', (err, data) => {
     if (err) {
       console.log(err);
       process.exit(2);
     }
 
-    const types = data.split('\n')
-      .map(line => line
-        .replace('/\* @flow \*/', '')
-        .replace(/,$/, ';')
-        .replace(/<\*>/g, '<any>')
-        .replace(': ?', '?: ')
-      ).join('\n');
+    const types = data
+      .split('\n')
+      .map(line =>
+        line
+          .replace('/* @flow */', '')
+          .replace(/,$/, ';')
+          .replace(/<\*>/g, '<any>')
+          .replace(': ?', '?: '),
```

Comment:
```suggestion
          .replace(': ?', '?: ')
```

That should fix the issue.